modules/36_buildings/simple/datainput.gms

```diff
@@ -92,19 +92,40 @@ if ((cm_ElLim_b lt 1),
 );
 
 
-*** FS: CES markup cost buildings
+*** CES mark-up cost buildings
```

Comment:
parts of this documentation should also show up in the automated documentation. Please, use "*`" for those parts

Reply:
done